iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ReactiveCapabilityCurveImpl.java

```diff
@@ -54,10 +58,42 @@ public double getMaxQ() {
 
     private final TreeMap<Double, Point> points;
 
-    ReactiveCapabilityCurveImpl(TreeMap<Double, Point> points) {
+    private static void checkPointsSize(TreeMap<Double, Point> points) {
         if (points.size() < 2) {
-            throw new IllegalStateException("Points size must be >= 2");
+            throw new IllegalStateException("points size should be >= 2");
+        }
+    }
+
+    private Point extrapolateReactiveLimitsSlope(double p, Point pBound) {
```

Comment:
I think it would be useful to share this method so it could be reused for custom IIDM implementations (for instance for powsybl-network-store).
It could be for instance in a utility class in the `com.powsybl.iidm.network.util` package of `iidm-api`.

The method should take a `TriFunction<Double, Double, Double, ReactiveCapabilityCurve.Point>` as parameter in order to create the result without knowing the used IIDM implementation.